### DISCUSSION TITLE
《汉兜》（Handle），供中国的 Wordler（Wordle 玩家）游玩

### DISCUSSION THREAD

**Discussion by Abelonx** [FloatTech/ZeroBot-Plugin#188] [Ideas]
功能名：《汉兜》（Handle），供中国的 Wordler（Wordle 玩家）游玩
功能具体：成语猜词
实现方法（可选）：项目地址，https://github.com/antfu/handle

**Comment by fumiama** [comment]
这个相对困难，可能不会很快实装。
